### ISSUE TITLE
Implementation of Power Spectral Density constraint

### ISSUE THREAD

**Issue by usryokousha** [rmr8019/psf-estimation#1]
I have implemented the same PSF framework in Matlab as well.  It seems that you had some trouble implementing the Power Spectral Density constraint from what I can see.  I used the derivative as specified in the Mosleh et Al paper, but my results were not consistent when checked against the numerical derivative.  Ideally, it would be great to get this working!

**Comment by roserustowicz** [comment]
@usryokousha yes, I was also having a hard time getting the analytical gradient with the PSD to match the numerical gradient.  I may revisit this soon, and I will let you know if I add in that portion of the derivative! In your case, did you see any improvement in the results even though your gradients did not check out perfectly?
